storage/columnstore/CMakeLists.txt

```diff
@@ -13,15 +13,17 @@ endmacro()
 
 IF(CMAKE_SYSTEM_PROCESSOR STREQUAL "x86_64" OR
 CMAKE_SYSTEM_PROCESSOR STREQUAL "amd64")
-    add_subdirectory(columnstore)
+    cmake_policy(SET CMP0093 NEW)
 
     # https://jira.mariadb.org/browse/MCOL-5611
-    FIND_PACKAGE(Boost 1.80.0 COMPONENTS system filesystem thread regex date_time chrono atomic)
-    IF (Boost_FOUND)
+    FIND_PACKAGE(Boost COMPONENTS system filesystem thread regex date_time chrono atomic)
+    IF (Boost_VERSION VERSION_GREATER "1.79.0")
```

Comment:
Nice switch from a specific target version to a range of versions that are not covered. Is there a roadmap to update the codebase to support Boost 1.80.0 and above?

Reply:
this fix is only for 10.5 which uses a very old ColumnStore. Using ColumnStore in 10.5 is generally not recommended. 10.6+ are ok